docs/source/custom_task.rst

```diff
@@ -4,26 +4,39 @@ Tutorial: Creating a Custom Task
 In this tutorial we will go over the process of creating a custom task,
 along with a custom data module.
 
+1 . Imports
+-----------
+
+
 .. testcode:: python
 
-    import flash
+    from typing import Any, List, Tuple
 
+    import numpy as np
     import torch
-    from torch.utils.data import TensorDataset, DataLoader
-    from torch import nn
+    from pytorch_lightning import seed_everything
     from sklearn import datasets
     from sklearn.model_selection import train_test_split
+    from torch import nn
+
+    import flash
+    from flash.data.auto_dataset import AutoDataset
+    from flash.data.process import Postprocess, Preprocess
+
+    seed_everything(42)
 
-The Task: Linear regression
----------------------------
+
+2 . The Task: Linear regression
+-------------------------------
 
 Here we create a basic linear regression task by subclassing
```

Comment:
Linear regression is a method. The task should be Forecasting or something.

Reply:
Renamed `RegressionTask`